Feature/integration test

mj610-kim opened the pull request:
Resolves #67

mj610-kim committed:
ci: add integration test CI workflow

Add integration test (run sample input and verify output) upon PR
opened, edited, and reopened.

mj610-kim committed:
ci: add integration test TCs

Add custom-made TCs:
- input_mj.txt, output_expected_mj.txt
- input_sj.txt, output_expected_sj.txt

Signed-off-by: 김명종 <mj610.kim@gmail.com>

seongjin-Park210 reviewed (approved):
검증이 철저하게 되겠네요! 이런것까지 추가하시다니 대단합니다!

kwonhj0116 commented on EmployeeManagementSystem/build.gradle:
```diff
@@ -1,9 +1,9 @@
 
-group 'org.example'
-version '1.0-SNAPSHOT'
+mainClassName = 'com.cra08.excellentcode.Main'
```
build.gradle 수정은 의도하신 건가요?!

mj610-kim replied:
의도는 했었는데, 삭제한 부분은 단순한 clean up이었습니다. 이 의도를 더 명확하게 하기 위해서 커밋을 두개로 분리했습니다. 확인 부탁드립니다!

kwonhj0116 reviewed (commented):
TC 가 풍부해졌네요! 고생하셨습니다~

mj610-kim committed:
refactor: remove unnecessary lines in gradle.build

EmployeeManagmentSystem does not need to separate from other gradle
environments by the 'group' property. Versioning is also handled by git
tags. Remove unused properties 'group' and 'version'.

Signed-off-by: 김명종 <mj610.kim@gmail.com>

mj610-kim committed:
ci: change integration test trigger to pull_request default

When head of PR branch is updated, another integration test should be
triggered. Change 'on' tag to default to include 'synchronize'.

Signed-off-by: 김명종 <mj610.kim@gmail.com>

taemin00 reviewed (approved):
와우 테스트를 이렇게 여러개를 할 수도 있군요! 정말 좋네요 ㅎㅎ
